common/nodes/database-schema.xml

```diff
@@ -220,6 +220,28 @@ CREATE TABLE rolls (
 	INDEX (Name)
 );
 
+<!-- Repos -->
+DROP TABLE IF EXISTS repos;
+CREATE TABLE repos (
+	id		INT AUTO_INCREMENT PRIMARY KEY,
+	name		VARCHAR(128) NOT NULL,
+	alias		VARCHAR(128) NOT NULL,
+	url		TEXT NOT NULL,
+	autorefresh	BOOLEAN DEFAULT FALSE,
+	assumeyes	BOOLEAN DEFAULT FALSE,
+	type		VARCHAR(32) NOT NULL DEFAULT 'rpm-md',
+	is_mirrorlist	BOOLEAN DEFAULT FALSE,
+	gpgcheck	BOOLEAN DEFAULT FALSE,
+	gpgkey		TEXT DEFAULT '',
+	os		VARCHAR(32) NOT NULL DEFAULT '&os;',
+	pallet_id	INT,
+	INDEX (Name),
+	INDEX (Alias),
+	FOREIGN KEY (pallet_id) REFERENCES rolls(id) ON DELETE CASCADE,
+	CONSTRAINT unique_repo_name UNIQUE (name),
+	CONSTRAINT unique_repo_alias UNIQUE (alias)
```

Comment:
You don't need to call out the UNIQUE constraint separately. You can put it in the field definition like so:
```
name		VARCHAR(128) NOT NULL UNIQUE,
alias		VARCHAR(128) NOT NULL UNIQUE
```